packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -91,6 +91,13 @@ interface EnqueueTaskParams {
   name: string;
 }
 
+/**
+ * Bot can throw this error to indicate it experienced some form of
+ * resource limitation.  GCFBootstrapper will catch this and return
+ * HTTP 503 to suggest Cloud Task adds backoff for the next attempt.
+ */
+export class ServiceUnavailable extends Error {}
```

Comment:
Should we wrap the underlying cause (stack) in this exception? We get the base error message from the constructor, but the stack could be useful.

Reply:
Now you can preserve the original stack trace and it will be in the log.